stories/Atom/Buttons-and-labels/Cta_button/buttons.stories.mdx

```diff
@@ -0,0 +1,200 @@
+  </div>
+</div>
+<div className="cta_wrapper">
+  <div className="cta_item">
+    <div className="cta__box cta-link-space"></div>
+    <div className="cta__label">Cta link with space expand </div>
+    <div className="cta__code"> <Ctaspace label="READ MORE"></Ctaspace></div>
+  </div>
+</div>
+
+<Story name="Primary-arrow">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Primarybuttonarrow label={caption.detail1} ></Primarybuttonarrow>;
+  }}
+</Story>
+
+<Story name="Primary arrow-disabled">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Primarybtnarrowdisbled label={caption.detail1} ></Primarybtnarrowdisbled>;
+  }}
+</Story>
+
+<Story name="Primary-no arrow">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Primarybuttonnoarrow label={caption.detail1} ></Primarybuttonnoarrow>;
+  }}
+</Story>
+
+<Story name="Primary no arrow-disabled">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Primarybtnnoarrowdisbled label={caption.detail1} ></Primarybtnnoarrowdisbled>;
+  }}
+</Story>
+
+<Story name="Secondary">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Secondarybutton label={caption.detail2} ></Secondarybutton>;
+  }}
+</Story>
+
+<Story name="Secondary-disabled">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Secondarybuttondisbled label={caption.detail2} ></Secondarybuttondisbled>;
+  }}
+</Story>
+
+<Story name="Cta-link-arrow">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Ctaarrow label={caption.detail1} ></Ctaarrow>;
+  }}
+</Story>
+
+<Story name="Cta-link-space">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Ctaspace label={caption.detail1} ></Ctaspace>;
+  }}
+</Story>
+
+<div className="subheading">Actions</div>
+<ul>
+  <li>When an action required by the user has more than one option, always use a a negative action button (secondary) paired with a positive action button (primary) in that order.</li>
+  <li>Negative action buttons will be on the left.</li>
+  <li>Positive action buttons should be on the right.</li>
+  <li>When these two types buttons are paired in the correct order, they will automatically space themselves apart.</li>
+</ul>
+
+<div className="subheading">Usage</div>
+<ul>
+  <li>We can take HTML from the HTML tab in the canvas tab</li>
+    <li>Include the common SCSS file like normalize, variables, mixins from the folder <code>assets/scss</code></li>
+    <li>Component wise SCSS file residing in each component<code>buttons.scss</code></li>
+    <li>Include the Compiled CSS file from the <code>assets/css/style.css</code></li>
+</ul>
```

Comment:
@pallavikadam-srijan Correct the documentation as mentioned by jharna for the previous tickets